doc/source/conf.py

```diff
@@ -62,7 +62,7 @@
 nbsphinx_allow_errors = True
 nbsphinx_kernel_name = 'python'
 numpydoc_show_class_members = False
-nbsphinx_timeout = 120
+nbsphinx_timeout = 360
```

Comment:
Is this required? The difference is not huge, but in my opinion it's better to leave a smaller timeout in the general configuration and only change the configuration of individual notebooks if they need to run longer (like I had to do for one of the vector fitting examples).

Reply:
This is the notebook I meant: https://raw.githubusercontent.com/scikit-rf/scikit-rf/master/doc/source/examples/vectorfitting/vectorfitting_Agilent_E5071B.ipynb
 At the very bottom, the timeout for this notebook is set to 240 seconds:
`"nbsphinx": {
   "timeout": 240
  }`

Reply:
You're right, it would be better to define this timeout for a single notebook. I'll do it.

Reply:
Is it actually the same vector fitting notebook that's causing a timeout error again? Because it has worked before when it got merged with the 240 s timeout.

Reply:
> 
> 
> Is it actually the same vector fitting notebook that's causing a timeout error again? Because it has worked before when it got merged with the 240 s timeout.

yes, it's the same:
```
vf.vector_fit(n_poles_real=1, n_poles_cmplx=26)
```
my machine is maybe is bit slower than yours ;)